massa-versioning-worker/src/versioning.rs

```diff
@@ -616,7 +635,10 @@ impl MipStoreRaw {
                     } else {
                         // Something is wrong (time range not ok? / version not incr? / names?
                         // or component version not incr?)
-                        should_merge = false;
+                        should_merge = Some(UpdateWithError::Overlapping(
```

Comment:
It's not super clear from the context what Overlapping means to me. Maybe give an example to make this clearer where this applies (here or in the Error enum definition)

Reply:
Added some examples